### PR TITLE
[WIP] Add new action, click each element in list and drag & drop

### PR DESCRIPTION
**Pull request by juliensz**
I add a new function for multi click on page :

- With a list passed in parameters
- With se same selector (like class selector ".button") 

I add to a drag and drop function.

### CODE REVIEW

**Comment by chris-bingham** [comment]
Nice, I was going to have to write a drag and drop step, thank you! :)

File: packages/browser.smash [hunk 1]
```diff
@@ -148,6 +148,20 @@
...
+        if(!Array.isArray(element)){
+            const elems = await $(element);
+            elems.forEach(elem => elem.click());
+        }
+        else {
+            const multiElems = element.forEach(elem => elem.click());
+        }
+    }
+
+    * Drag {{dragPosition}} in {{dropPosition}} {
```
Comment (vptes1) [inline packages/browser.smash]: I would reword this as 
`Drag {{elementFrom}} to {{elementTo}}` and `Drag {{elementFrom}} to {{x}} {{y}}` (2 different functions)

See `dragAndDrop()` docs: https://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/lib/input_exports_Actions.html

File: packages/browser.smash [hunk 2]
```diff
@@ -148,6 +148,20 @@
         }
     }
 
+    * Click each {{element}} {
+        if(!Array.isArray(element)){
+            const elems = await $(element);
```
[the remark this answers is not in the document]
Comment (vptes1) [reply packages/browser.smash]: Also, I don't really envision many users passing in an array, as everything in smashtest is a string

File: packages/browser.smash (hunk 1 above)
Comment (juliensz) [reply packages/browser.smash]: Ok i can reword this and create 2 different functions.

File: packages/browser.smash (hunk 2 above)
[the remark this answers is not in the document]
Comment (juliensz) [reply packages/browser.smash]: i set a array if people need to set array with multiple element haven't a same Id or class

[the remark this answers is not in the document]
Comment (vptes1) [reply packages/browser.smash]: But it won't be easy for them to set an array. They'd have to set it in a code block using something like `g()`.
```
Click each [one, two] // NOT an array - this is the string 'one, two'
```
Easier just to use multiple `Click` steps in succession.

[the remark this answers is not in the document]
Comment (juliensz) [reply packages/browser.smash]: Oh ! nice I did not think about doing it like that but it's a great idea a table disguised to make several clicks on items of different name

**Comment by juliensz** [comment]
Not a final function i work on it, i push so that you can see where I am

File: packages/browser.smash [hunk 3]
```diff
@@ -148,6 +148,20 @@
...
+        const elems = await $$(list);
+        elems.forEach(elem => elem.click());
+    }
+
+    * Drag {{elementFrom}} in {{elementTo}} {
+        await browser.driver.actions({bridge: true}).dragAndDrop(elementFrom, elementTo).perform();
+    }
+
+    * Drag {{elementFrom}} to {{x}}, {{y}} {
+        await browser.driver.actions({bridge: true}).dragAndDrop(dragPosition, {x: parseInt(x), y: parseInt(y)}).perform();
```
Comment (vptes1) [inline packages/browser.smash]: Variable `dragPosition` doesn't exist. A unit test would catch that :)

File: packages/browser.smash [hunk 4]
```diff
@@ -148,6 +148,20 @@
         }
     }
 
+    * Click each {{element}} {
+        const list = element.split(",");
+        const elems = await $$(list);
+        elems.forEach(elem => elem.click());
+    }
+
+    * Drag {{elementFrom}} in {{elementTo}} {
```
Comment (vptes1) [inline packages/browser.smash]: in --> to

File: packages/browser.smash [hunk 5]
```diff
@@ -148,6 +148,20 @@
         }
     }
 
+    * Click each {{element}} {
+        const list = element.split(",");
+        const elems = await $$(list);
```
Comment (vptes1) [inline packages/browser.smash]: No this isn't right. You had it right before. You shouldn't take a comma-separated list (among many things, the comma is a special character in an EF and this will interfere with that). You just `$$(element)` and do the `forEach()` below.

Comment (juliensz) [reply packages/browser.smash]: Ok my bad

File: packages/browser.smash (hunk 4 above)
Comment (juliensz) [reply packages/browser.smash]: Its good now

File: packages/browser.smash (hunk 3 above)
Comment (juliensz) [reply packages/browser.smash]: Sorry iu didn't test this :)

**Comment by juliensz** [comment]
yeah i write this test
